src/plugins/marks/index.tsx

```diff
@@ -584,6 +602,37 @@ export class MarksPlugin {
 
     return null;
   }
+
```

Comment:
should it be just combined into a single function that takes in the cursor column?

Reply:
this way is also fine if you think they might be used in isolation

Reply:
I think it might be useful this way later. I'm working on mark autocompletion now.